defaults/main.yml

```diff
@@ -3,6 +3,6 @@
 # Here is the right place to put the role's input variables.
 # This file also serves as a documentation for such a variables.
 
-# Examples of role input variables:
-template_foo: "foo"
-template_bar: "bar"
+crypto_policies_policy: false
```

Comment:
This is a default saying that we do not want to set policy to anything, just report what is the current and available policies (see a related comment below). The other option would be not defining it here (not recommended in best practices?) or use separate variables (more cumbersome for users?).

Reply:
I think this is fine.

Reply:
But this default should be `null`, not `false`.